packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/RequestBodyUtil.java

```diff
@@ -152,7 +151,7 @@ public void writeTo(BufferedSink sink) throws IOException {
           source = Okio.source(inputStream);
           sink.writeAll(source);
         } finally {
-          Util.closeQuietly(source);
+          source.close();
```

Comment:
Can we copy over the code of `closeQuitely`, namely:
```kotlin
fun Closeable.closeQuietly() {
  try {
    close()
  } catch (rethrown: RuntimeException) {
    throw rethrown
  } catch (_: Exception) {
  }
}
```

Reply:
thanks for the suggestion, i’ll update it later